### DISCUSSION TITLE
Level 3 NUTS

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#558] [Q&A] [answered]
In the eForms, the level of detail of the NUTs that are sent, according to the validations, is level 3.

If, for example, we want to send level 2 NUTs (because the tender affects a broader region, for example, an autonomous community in the case of Spain). Should we always send all the regions (level 3 NUTs) that make up that autonomous community?

For example, if the tender is for the Basque government that covers the entire Basque country?

ES21 -> Basque Country
ES211 -> Araba/Álava
ES212 -> Gipuzkoa
ES213 -> Bizkaia

Another doubt about the NUTs. If a file was sent from Ted with level 2 NUTs, what will happen when sending the eForm associated with that file when it is now sent with level 3 NUTs?

Thanks in advance,
Alex

**Comment by ferrakaop** [comment, marked as answer]
The technical answer is that only NUTS3 codes can be included in the XML, e.g. one or more of these values for Spain: https://docs.ted.europa.eu/eforms/latest/reference/code-lists/nuts-esp-lvl3.html

The eForms regulation requires NUTS at this level so it's up to the front-end to perhaps make it easier for end users to select multiple NUTS3 codes to make up a bigger region.

Your second question is more business related: if your earlier notice (e.g. contract notice) was in current TED-XML with NUTS2, and your next notice (e.g. contract award notice) is in eForms, then you'll need to put NUTS3 in your CAN, even if you only specified NUTS2 in your CN.  

This sort of scenario will also happen if you change a notice that overlaps your switch to eForms: the new Change notice will need to comply with the eForms rules and fields (like NUTS3), even if some information didn't exist in the original TED-XML notice (only had NUTS2).
